### PR TITLE
doc/contributing: info about where the docs are located

### PR DESCRIPTION
**Pull request by camilamacedo86**
#### What this PR does / why we need it:

Let contributors know where the docs are located.

#### Which issue(s) this PR fixes:
NONE
#### Special notes for your reviewer:
NONE
## Steps to reproduce
NONE
#### Does this PR introduce a user-facing change?
NONE#### Does this PR require documentation?
NONE

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -238,3 +238,8 @@ Snapshot management requires VM t be powered off.
 ### QEMU on MacOS
 
 You might want try to use [UTM](https://mac.getutm.app/).
+
+## Documentation
+
+If you would like to contribute with the docs published in https://kurl.sh/docs/introduction/ then,
+see that they are built from the repository [replicatedhq/kurl.sh][kulr-docs].
```
Comment (emosbaugh) [inline CONTRIBUTING.md:245]: do you have this target already? `kulr-docs` also it is misspelled

Comment (camilamacedo86) [reply CONTRIBUTING.md]: Sorry, it came from the split. 
I added all links with alias but in the review was requested not to use alias. 
Fixing that now.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -239,6 +239,11 @@ Snapshot management requires VM t be powered off.
 
 You might want try to use [UTM](https://mac.getutm.app/).
 
+## Documentation
+
+If you would like to contribute with the docs published in https://kurl.sh/docs/introduction/ then,
+see that they are built from the repository [replicatedhq/kurl.sh][kulr-docs].
```
Comment (camilamacedo86) [inline CONTRIBUTING.md:245]:
```suggestion
see that they are built from the repository [replicatedhq/kurl.sh](https://github.com/replicatedhq/kurl.sh).
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -247,6 +247,11 @@ Snapshot management requires VM t be powered off.
 
 You might want try to use [UTM](https://mac.getutm.app/).
 
+## Documentation
+
+If you would like to contribute with the docs published in https://kurl.sh/docs/introduction/ then,
+see that they are built from the repository [replicatedhq/kurl.sh](https://github.com/replicatedhq/kurl.sh).
```
Comment (emosbaugh) [inline CONTRIBUTING.md:253]:
```suggestion

For contributions to the kURL documentation published at https://kurl.sh/docs/introduction/,
see the source repository at [replicatedhq/kurl.sh](https://github.com/replicatedhq/kurl.sh).
```

**Review by emosbaugh — CHANGES_REQUESTED** [review]

**Review by emosbaugh — DISMISSED** [review]

**Review by emosbaugh — APPROVED** [review]

**Pull request merged**